src/python/grpcio/grpc/_server.py

```diff
@@ -745,11 +758,14 @@ def add_insecure_port(self, address):
   def add_secure_port(self, address, server_credentials):
     return _add_secure_port(self._state, _common.encode(address), server_credentials)
 
+  def add_shutdown_handler(self, handler):
+    self._state.shutdown_handlers.append(handler)
+
   def start(self):
     _start(self._state)
 
-  def stop(self, grace):
-    return _stop(self._state, grace)
+  def stop(self, grace, shutdown_handler_grace=None):
```

Comment:
nit-question: any particular reason why this is called `shutdown_handler_grace` and other places call it `exit_handler_grace`?